tests/behat/features/capabilities/custom/post_request_pattern.feature

```diff
@@ -0,0 +1,193 @@
+Feature: Test REST API POST request
+  In order create a pattern
+  As LIU
```

Comment:
What does LIU mean?

Reply:
Oh, sorry for the shortcut. Its stands for Logged In User.